src/lib/render-lit-html.ts

```diff
@@ -181,7 +181,7 @@ export async function* renderValue(
         const templateResult = (instance.instance as any).renderLight();
         yield* renderValue(templateResult, childRenderer, renderInfo);
       }
-    } else if (value === nothing) {
+    } else if (value === nothing || value === noChange) {
```

Comment:
When is it the case that the value here is `noChange`. Add a comment?

Reply:
It's a value users can set like any other.